suite-native/device/src/hooks/useDeviceLowBatteryAlert.tsx

```diff
@@ -0,0 +1,35 @@
+import { useCallback } from 'react';
+import { useSelector } from 'react-redux';
+
+import {
+    DEVICE_LOW_BATTERY_PERCENTAGE_THRESHOLD,
+    selectIsDeviceConnectedViaBluetooth,
+    selectIsDeviceLowOnBattery,
+} from '@suite-common/wallet-core';
+import { useAlert } from '@suite-native/alerts';
+import { Translation } from '@suite-native/intl';
+
+export function useDeviceLowBatteryAlert() {
+    const isDeviceConnectedViaBluetooth = useSelector(selectIsDeviceConnectedViaBluetooth);
+    const isDeviceLowOnBattery = useSelector(selectIsDeviceLowOnBattery);
+    const { showAlert } = useAlert();
+
+    const showBatteryAlert = useCallback(() => {
+        showAlert({
+            title: <Translation id="bluetooth.alerts.lowBattery.title" />,
+            description: (
+                <Translation
+                    id="bluetooth.alerts.lowBattery.description"
+                    values={{ percentage: DEVICE_LOW_BATTERY_PERCENTAGE_THRESHOLD }}
+                />
+            ),
+            primaryButtonTitle: <Translation id="generic.buttons.gotIt" />,
+            primaryButtonVariant: 'redBold',
+        });
+    }, [showAlert]);
+
+    return {
+        shouldShowAlert: isDeviceConnectedViaBluetooth && isDeviceLowOnBattery,
```

Comment:
nit: for better readability
```suggestion
        shouldShowBatteryAlert: isDeviceConnectedViaBluetooth && isDeviceLowOnBattery,
```

Reply:
good nit!